src/components/dialogs/incorporate-now-error.vue

```diff
@@ -32,8 +32,11 @@ export default class IncorporateNowErrorDialog extends Vue {
   /** Prop to provide attachment selector. */
   @Prop() readonly attach: string
 
-  /** Pass click event to parent. */
-  @Emit() close () { }
+  /** Pass click event to parent. Remove the session storage LEGAL_TYPE variable. */
+  @Emit()
+  close () {
+    sessionStorage.removeItem('LEGAL_TYPE')
```

Comment:
To keep this dialog naive, I think the parent should clear the session storage.

Reply:
-> "Simple components" that should not know the big picture or business logic. This component should just be an error dialog, with the work done elsewhere.

Reply:
Makes a lot of sense. On it Sev.

Reply:
Looks good.